### PR TITLE
DamarisWriter: some additional cleanup

### PR DESCRIPTION
**Pull request by akva2**
Cosmetics and some code shuffling.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: ebos/damariswriter.cc [hunk 1]
```diff
@@ -81,4 +85,62 @@ int endIteration(int rank)
     return dam_err;
 }
 
+int setupWritingPars(Parallel::Communication comm,
+                     const int n_elements_local_grid,
+                     std::vector<unsigned long long>& elements_rank_offsets)
+{
+    // one for each rank -- to be gathered from each client rank
+    std::vector<unsigned long long> elements_rank_sizes(comm.size());
```
Comment (bska) [inline ebos/damariswriter.cc:93]: I guess we get a declaration of `vector<>` from somewhere else, but we really should `#include <vector>` here too.

**Review by bska — APPROVED** [review]
Apart from a missing `#include` statement, this looks good to me.

File: ebos/damariswriter.cc (hunk 1 above)
Comment (akva2) [reply ebos/damariswriter.cc:93]: indeed. i thought it was in the header but it's not. added there.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**